### PR TITLE
Add support opening subdirs

### PR DESCRIPTION
**Pull request by martinRenou**
Follow up https://github.com/jtpio/jupyterlab-filesystem-access/pull/1

subdirs

### CODE REVIEW

File: src/drive.ts [hunk 1]
```diff
@@ -1,7 +1,21 @@
 import { Contents, ServerConnection } from '@jupyterlab/services';
 
+import { PathExt } from '@jupyterlab/coreutils';
+
 import { ISignal, Signal } from '@lumino/signaling';
 
+async function toArray<T>(
```
Comment (jtpio) [inline src/drive.ts:7]: Wondering if the one from Lumino would work here?

```ts
import { toArray } from '@lumino/algorithm';
```

Comment (jtpio) [reply src/drive.ts:7]: If so we can then add `@lumino/algorithm` to the `dependencies` in `package.json`

Comment (martinRenou) [reply src/drive.ts:7]: I didn't know this was a thing, thanks! :)

Comment (martinRenou) [reply src/drive.ts:7]: Ah actually this does not work as the `toArray` I introduced takes and `AsyncIterableIterator` as input, while Lumino's takes an `ArrayLike` which is incompatible.

Comment (jtpio) [reply src/drive.ts:7]: Ah ok thanks for checking :+1:

**Comment by jtpio** [comment]
Thanks!

**Comment by jtpio** [comment]
Making a new release with this now.

**Pull request merged**